### PR TITLE
Vesting test

### PR DESCRIPTION
**Pull request by jatinj615**
UnrealTokenVesting contract test cases in ethers.js

### CODE REVIEW

File: test/sample-test.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+// const Token = artifacts.require("./contracts/UniLendToken.sol")
```
Comment (unrealfi) [inline test/sample-test.js]: Please delete not required files.

File: test/vesting.test.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    
+    it("should test token vesting for amount exactly equal to the balance of vesting contract", async function() {
+        let p = []
+        // Time travel
+        let second = 10000 * 1560 * 60
+        await increaseBlockTime(second)
+        await mineOneBlock()
+        // change num_vesting to number of vestings.
+        let num_vesting = 1
+        // Comment out below code if num_vesting > 1
```
Comment (unrealfi) [inline test/vesting.test.js]: Can we do a round of refactor here?

**Review by unrealfi — CHANGES_REQUESTED** [review]
Please also run lint.

File: test/vesting.test.js (hunk 2 above)
Comment (jatinj615) [reply test/vesting.test.js]: Done

**Review by unrealfi — APPROVED** [review]
LGTM! Amazing work @jatinj615!

**Pull request merged**